### DISCUSSION TITLE
Still the auto completion with MSSQL for dbo tables in 5.3.3 -- ommiting owner in query

### DISCUSSION THREAD

**Discussion by mamendex** [dbeaver/dbeaver#5078] [Ideas]
Hi,

I'm trying to use DBeaver instead of SQL Server Management Studio since I work with MSSQL, Oracle and Mysql databases.

But I have a lot of queries already prepared using the sintax:

`> 	select * from nus..C_USER s where s.ide_usr='148'`

With which DBeaver seems not to work to guess the "ide_usr" from "s" table. 
If I make explicit the owner of the table:

`> 	select * from nus.dbo.C_USER s where s.ide_usr='148'`

The Completion works as expected.

Of course, it is not a lot of work to replace all ".." with ".dbo."; but, since SQL Server Management Studio works well with this sintax, I ask: is that an issue, maybe a misconfiguration of my connection? If not, wouldn't that be a nice-to-have feature?

**Comment by LonwoLonwo** [comment]
Hello @mamendex 
Thanks for the suggestion. But auto-complete to the SQL Editor is universal for different databases, and different databases may have different schemas indicated after the name of the database. Not only "dbo". 
Therefore, we hope that users will use the full path to the object with the schema. 
And we want to avoid misunderstandings if we guess the schema name that is missing between the object name and the database name.
